test/datasets.jl

```diff
@@ -5,11 +5,20 @@ end
 
 using NetCDF, Zarr
 
-@testset "Reading NetCDF" begin
+using Pkg.Artifacts
 import Downloads
-p = Downloads.download("https://www.unidata.ucar.edu/software/netcdf/examples/sresa1b_ncar_ccsm3-example.nc")
+# This is the path to the Artifacts.toml we will manipulate
+artifact_toml =  "/home/fgans/julia_dev/YAXArrayBase/test/Artifacts.toml"
```

Comment:
This should be a relative path inside the package, otherwise the CI is going to complain.